src/antsibull/cli/antsibull_build.py

```diff
@@ -345,6 +356,25 @@ def parse_args(program_name: str, args: List[str]) -> argparse.Namespace:
                                help='Path to a ansible_collections directory containing a'
                                ' collection tree to check.')
 
+    validate_tags = subparsers.add_parser('validate-tags',
+                                          parents=[build_parser])
+    validate_tags.add_argument(
+        '--deps-file',
+        default=None,
+        help='File which contains the list of collections and'
+        ' versions which were included in this version of Ansible.'
+        '  This is considered to be relative to --data-dir.'
+        f'  The default is {DEFAULT_FILE_BASE}-X.Y.Z.deps',
+    )
+    tag_file = validate_tags.add_mutually_exclusive_group()
+    tag_file.add_argument('-i',
+                          '--input',
+                          help=('Collection tag data file to validate.'
+                                'Mutually exclusive with --output.'))
+    tag_file.add_argument('-o',
+                          '--output',
+                          help='Path to output a collection tag data file.'
+                          'If this is ommited, no tag data will be written')
```

Comment:
Hmm, how about simplifying the interface by having two new subcommands instead of one: `validate-tags` (which is basically what you have now minus `--input`) and `validate-tags-file` (which is `--input`). What do you think?

Reply:
I like it!